dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/export/trackedentity/TrackedEntityChangeLogServiceTest.java

```diff
@@ -209,16 +227,20 @@ void shouldReturnChangeLogsWhenTrackedEntityAttributeValueIsCreated() throws Not
     programOwnerService.createOrUpdateTrackedEntityProgramOwner(
         trackedEntity.getUid(), program.getUid(), orgUnitA.getUid());
 
+    createAndPersistProgramAttribute(program);
+
     Page<TrackedEntityChangeLog> changeLogs =
         trackedEntityChangeLogService.getTrackedEntityChangeLog(
-            UID.of(trackedEntity.getUid()), UID.of(program.getUid()), defaultPageParams);
+            UID.of(trackedEntity.getUid()),
+            UID.of(program.getUid()),
+            defaultOperationParams,
+            defaultPageParams);
 
     assertNumberOfChanges(1, changeLogs.getItems());
     assertAll(
         () ->
             assertChange(
```

Comment:
can you please use/add the assertCreate/assertUpdate/Delete assertions we have for event changelogs. it makes the tests easier to read.